src/rust/cryptography-x509/src/extensions.rs

```diff
@@ -312,6 +312,20 @@ pub struct ProfessionInfo<'a> {
     pub add_profession_info: Option<&'a [u8]>,
 }
 
+// #[derive(asn1::Asn1Read, asn1::Asn1Write)]
+pub struct Admission<'a> {
+    // #[explicit(0)]
+    pub admission_authority: Option<name::GeneralName<'a>>,
+    // #[explicit(1)]
+    pub naming_authority: Option<NamingAuthority<'a>>,
+    /*
+    pub profession_infos: common::Asn1ReadableOrWritable<
+        asn1::SequenceOf<'a, ProfessionInfo<'a>>,
+        asn1::SequenceOfWriter<'a, ProfessionInfo<'a>, Vec<ProfessionInfo<'a>>>,
+    >,
```

Comment:
I assume this needs to be commented out because the traits are not yet available?

Reply:
@alex yes exactly, the `SequenceOfWriter` expects the Asn1Readable/Writable traits, orherwise the code will not compile.

Reply:
I guess this is fine for now.